api/apps/sdk/retrieval.py

```diff
@@ -0,0 +1,62 @@
+from flask import request, jsonify
+
+from db import LLMType, ParserType
+from db.services.knowledgebase_service import KnowledgebaseService
+from db.services.llm_service import LLMBundle
+from settings import retrievaler, kg_retrievaler, RetCode
+from utils.api_utils import validate_request, build_error_result, apikey_required
+
+
+@manager.route('/retrieval', methods=['POST'])
```

Comment:
```suggestion
@manager.route('/dify_retrieval', methods=['POST'])
```

Reply:
<img width="899" alt="image" src="https://github.com/user-attachments/assets/e3434ef3-0f76-431e-afc6-141b0e3ad8f9">
Thanks for your comment buy the api request url needs to be "endpoint/retrieval".
https://docs.dify.ai/guides/knowledge-base/external-knowledge-api-documentation

Reply:
I've added dify to url prefix and changed the filename, PTAL.